internal/services/oracle/oracle_autonomous_database_clone_resource.go

```diff
@@ -0,0 +1,683 @@
+	AutoScalingForStorageEnabled bool    `tfschema:"auto_scaling_for_storage_enabled"`
+	MtlsConnectionRequired       bool    `tfschema:"mtls_connection_required"`
+	NationalCharacterSet         string  `tfschema:"national_character_set"`
+	SubnetId                     string  `tfschema:"subnet_id"`
+	VnetId                       string  `tfschema:"virtual_network_id"`
+
+	// Optional for Clone
+
+	CustomerContacts        []string `tfschema:"customer_contacts"`
+	RefreshableModel        string   `tfschema:"refreshable_model"`
+	TimeUntilReconnectClone string   `tfschema:"time_until_reconnect_clone"`
+
+	// optional for clone from backup timestamp
+
+	Timestamp                         string `tfschema:"timestamp"`
+	UseLatestAvailableBackupTimeStamp bool   `tfschema:"use_latest_available_backup_time_stamp"`
+}
+
+func (AutonomousDatabaseCloneResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.Location(),
+
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+			ForceNew:     true,
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		// Clone-specific required fields
+
+		"source": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(autonomousdatabases.SourceTypeBackupFromTimestamp),
+				string(autonomousdatabases.SourceTypeBackupFromId),
+				string(autonomousdatabases.SourceTypeDatabase),
+				string(autonomousdatabases.SourceTypeCloneToRefreshable),
+			}, false),
+			DiffSuppressFunc: func(k, old, new string, d *pluginsdk.ResourceData) bool {
+				// Source is create-only and not returned by Azure API
+				return old != "" && new == ""
+			},
+		},
+		"source_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: autonomousdatabases.ValidateAutonomousDatabaseID,
+		},
+
+		"clone_type": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(autonomousdatabases.CloneTypeFull),
+				string(autonomousdatabases.CloneTypeMetadata),
+			}, false),
+		},
+		"data_base_type": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(autonomousdatabases.DataBaseTypeClone),
+				string(autonomousdatabases.DataBaseTypeCloneFromBackupTimestamp),
+			}, false),
+		},
+
+		// optional
+
+		"refreshable_model": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(autonomousdatabases.RefreshableModelTypeAutomatic),
+				string(autonomousdatabases.RefreshableModelTypeManual),
+			}, false),
+		},
+
+		"time_until_reconnect_clone": {
```

Comment:
Since this is a `database_clone` resource, `clone` is probably redundant, this also sounds like it's just a computed property?

If this can be set by the user, please add it to a test

```suggestion
		"time_until_reconnect": {
```